client/components/filters/date/style.scss

```diff
@@ -37,11 +37,17 @@
 .woocommerce-filters-date__tab {
 	outline: none;
 	border: 1px solid $woocommerce;
-	padding: 10px;
+	padding: 9px;
```

Comment:
We have some variables for padding/spacing: 

https://github.com/woocommerce/wc-admin/blob/master/client/stylesheets/abstracts/_variables.scss

Should this be `$gap-smaller` (8px) instead of 9px?

Reply:
Sounds good - I'm also going to add a `$gap-larger` at 36px to address the comment from @ryelle below